---
title: "Analysis of CRUK cfDNA run on the _DLCBL-2021.v1_ panel"
date: "\today"
output:
  pdf_document:
    keep_tex: yes
    extra_dependencies: float, fancyhdr
    
---

This is a report generated by `knitr::spin()`.

$\usepackage{fancyhdr}$
$\pagestyle{fancy}$
$\fancyhead[LE,RO]{Laura}$
Oxford Molecular Diagnostic Centre ![This is a figure.](images/oxford-molecular-diagnostics-centre.png)
knitr::opts_chunk$set(fig.pos = "!H", out.extra = "")
Department of Oncology
Oxford University
Level 4
John Radcliffe Hospital
Oxford OX3 9DU


Let's try some **knitr** options:
```{r echo=FALSE}
library(tidyverse)
library(ggplot2)
library(readr)
library(dplyr)

```


```{r echo=FALSE, fig.width=7}
#Import vcf
vcf<- read.csv ("/Users/lauralopezpascua/Analysis/DLBLC/CRUK/varscan/subtractions_edit/SCR16508-Baseline-c-0.varscan.annotation.tsv",sep="\t")
View(vcf)
#apply ACGM filters <3 reads,<5% MAF, remove synonimous
vcf<- filter(vcf, `Alt.Depth` > 2)
vcf<- filter(vcf, `MAF` < 0.01 | is.na(MAF))
vcf <-filter(vcf, Consequences != 'synonymous_variant' | is.na(Consequences))
vcf <-filter(vcf, Consequences != 'intron_variant' | is.na(Consequences))
likely_pathogenic <-filter(vcf, Sift != 'tolerated' , Polyphen != 'benign')
View(likely_pathogenic)
plot(cars)
```
Average deduplicated depth at variant call = ``r  mean(vcf$Depth)``

Now write an inline value. We know the value of $\pi$ is
``r  pi  ``
.

Finally please note that all roxygen comments are
optional. You do not need chunk options, either,
unless you want more control over the output
elements such as the size of plots.

Roses are \textcolor{red}{red}, violets are \textcolor{blue}{blue}.