---
title: "Analysis of CRUK cfDNA run on the _DLCBL-2021.v1_ panel"
date: \today
output:
  pdf_document:
    keep_tex: true
---

This is a report generated by `knitr::spin()`.

Let's try some **knitr** options:
```{r echo=FALSE}
library(tidyverse)
library(ggplot2)
library(readr)

```


```{r echo=FALSE, fig.width=7}
#Import vcf
vcf<- read.csv ("/Users/lauralopezpascua/Analysis/DLBLC/CRUK/varscan/subtractions_edit/SCR16508-Baseline-c-0.varscan.annotation.tsv",sep="\t")
#apply ACGM filters <3 reads,<5% MAF, remove synonimous
vcf<- filter(vcf, `Alt.Depth` > 2)
vcf<- filter(vcf, `MAF` < 5 | is.na(MAF))
vcf <-filter(vcf, Consequences != 'synonymous_variant' | is.na(Consequences))
vcf <-filter(vcf, Consequences != 'intron_variant' | is.na(Consequences))
likely_pathogenic <-filter(vcf, Sift != 'tolerated' , Polyphen != 'benign')
View(likely_pathogenic)
plot(cars)
```
Average deduplicated depth at variant call = ``r  mean(vcf$Depth)``

Now write an inline value. We know the value of $\pi$ is
``r  pi  ``
.

Finally please note that all roxygen comments are
optional. You do not need chunk options, either,
unless you want more control over the output
elements such as the size of plots.

Roses are \textcolor{red}{red}, violets are \textcolor{blue}{blue}.